pipelines/src/main/scala/dagr/pipelines/CreateUnmappedBamFromFastqPipeline.scala

```diff
@@ -53,7 +53,7 @@ class CreateUnmappedBamFromFastqPipeline(
   @Arg(doc="Path to a temporary directory.")                                      val tmp: Path,
   @Arg(flag="o", doc="The output prefix for files that are kept.")                val output: Path,
   @Arg(doc="Path to the unmapped BAM.  Use the output prefix if none is given. ") var unmappedBam: Option[PathToBam] = None
-) extends Pipeline {
+) extends Pipeline(Some(output.toAbsolutePath.getParent)) {
```

Comment:
Perhaps we should make an implicit method on a `Path`: `getAbsoluteParentPath`?  Your call.